src/spdx2/agent/spdx2.php

```diff
@@ -436,8 +436,12 @@ protected function addScannerResults(&$filesWithLicenses, ItemTreeBounds $itemTr
     {
       $reportedLicenseId = $this->licenseMap->getProjectedId($row['rf_fk']);
       $shortName = $this->licenseMap->getProjectedShortname($reportedLicenseId);
-      if ($shortName != 'No_license_found' && $shortName != 'Void') {
-        $filesWithLicenses[$row['uploadtree_pk']]['scanner'][] = $shortName;
+      if ($shortName != 'Void') {
+        if($shortName != 'No_license_found'){
+          $filesWithLicenses[$row['uploadtree_pk']]['scanner'][] = $shortName;
+        }else{
+          $filesWithLicenses[$row['uploadtree_pk']]['scanner'][] = "";
```

Comment:
@maximilianhuber 
- if the scanners aren't able to find any license then `LicenseInfoInFile: NONE` is accurate.
It is ok to say `NOASSERTION`, but the preference would be to provide the information that no information was found in the file by using `NONE`.  see comment on issue